### PR TITLE
Chore: Xcode 11

### PR DESCRIPTION
**Pull request by billypchan**
Update Xcode on Azure to Xcode 11.4.1 and use macOS 11.15
Update Xcode on Jenkins to Xcode 11.4.1

### CODE REVIEW

File: Jenkins/UI/Playground.groovy [hunk 1]
```diff
@@ -3,8 +3,13 @@ pipeline {
...
         ansiColor('xterm')
     }
+    
+    environment { 
+        DEVELOPER_DIR = "/Applications/Xcode_11.4.app/Contents/Developer/"
+    }
+    
     parameters {
-        string(defaultValue: "chore/xcode11.3", description: 'Branch to use', name: 'branch_to_build')
+        string(defaultValue: "chore/xcode11.4", description: 'Branch to use', name: 'branch_to_build')
```
Comment (johnxnguyen) [inline Jenkins/UI/Playground.groovy:12]: Is this supposed to be 11.4 or 11.3?

Comment (billypchan) [reply Jenkins/UI/Playground.groovy:12]: yes, and this default value will be removed after upgrading.

Comment (billypchan) [reply Jenkins/UI/Playground.groovy]: removed.

**Comment by typfel** [comment]
Just checking, are you planning merging this now? Because that would mean we can't make any releases until the Xcode 11 transition is done.

**Comment by billypchan** [comment]
I still need to check all framework can be build with Xcode 11.4. We can merge this at the end of this week.

File: Azure/framework-pipelines.yml [hunk 2]
```diff
@@ -1,17 +1,17 @@
 jobs:
   - job: Test
     pool:
-      vmImage: "macOS 10.14"
+      vmImage: "macOS-10.15"
     steps:
     - template: build-and-test/build-and-test.yml
       parameters:
-        branch: master
+        branch: chore/xcode11.3
```
Comment (billypchan) [inline Azure/framework-pipelines.yml:8]:
```suggestion
        branch: master
```

File: Azure/framework-pipelines.yml [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
     pool:
-      vmImage: "macOS 10.14"
+      vmImage: "macOS-10.15"
     dependsOn: Test
     condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/'))
     steps:
     - template: release-framework/release-framework.yml
       parameters:
-        branch: master
+        branch: chore/xcode11.3
```
Comment (billypchan) [inline Azure/framework-pipelines.yml:17]:
```suggestion
        branch: master
```

File: Azure/release-framework/release-framework.yml [hunk 4]
```diff
@@ -1,12 +1,12 @@
 parameters:
-  branch: "master"
+  branch: "chore/xcode11.3"
```
Comment (billypchan) [inline Azure/release-framework/release-framework.yml:2]:
```suggestion
  branch: "master"
```

**Pull request merged**